README.md

```diff
@@ -100,7 +100,7 @@ Then run the following commands.
 
 ```sh
 bundle install
-rake db:migrate
+rails db:setup db:migrate
```

Comment:
Actually, I think these are mutually exclusive. If you're setting up a new environment `db:setup` makes sense. If you're working with an existing environment, you would want to `db:migrate` instead.

```suggestion
rails db:setup
```